### PR TITLE
updated gradles

### PR DESCRIPTION
**Pull request by TacoTheDank**
**What does this PR do?**

Updates some dependencies without having to change any of the actual code. Put in some lines for future upgrades to the rxjava library. Renamed "Rx_lint_version" to "RX_LINT_VERSION" to match the rest of them.

**Database changed?**

No

**Where should the reviewer start?**

the main gradle

**How should this be manually tested?**

by building

**What are the relevant tickets?**

not sure if theres any relevant tickets

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.) 

I'm pretty sure not

**Code Review Checklist**

- [ ] Architecture
- [ ] Documentation on public interfaces
- [ ] Database changed?
- [ ] If yes - Database migration?
- [x] Remove comments & unused code & forgotten testing Logs
- [ ] Codestyle
- [ ] Partners build
- [x] Unit tests pass
- [x] Functional QA tests pass

### CODE REVIEW

File: crashreports/build.gradle [hunk 1]
```diff
@@ -45,13 +45,15 @@ dependencies {
...
-  androidTestImplementation("com.android.support.test.espresso:espresso-core:2.2.2", {
+  androidTestImplementation("com.android.support.test.espresso:espresso-core:3.0.2", {
     exclude group: 'com.android.support', module: 'support-annotations'
   })
   testImplementation "junit:junit:4.12"
   implementation "io.reactivex:rxjava:${RXJAVA_VERSION}"
+  // implementation "io.reactivex.rxjava2:rxjava:${RXJAVA2_VERSION}"
   implementation "io.reactivex:rxandroid:${RXANDROID_VERSION}"
-  implementation "nl.littlerobots.rxlint:rxlint:${Rx_lint_version}"
+  implementation "io.reactivex.rxjava2:rxandroid:${RXANDROID_VERSION}"
```
Comment (jdandrade) [inline crashreports/build.gradle]: This will try to resolve io.reactivex.rxjava2:rxandroid:1.2.1 ; It won't compile.

Comment (TacoTheDank) [reply crashreports/build.gradle]: should be fixed now

**Comment by TacoTheDank** [comment]
should be fixed now

**Comment by TacoTheDank** [comment]
:thonk:

**Comment by TacoTheDank** [comment]
this should work for now, i hope

**Comment by TacoTheDank** [comment]
gonna do this through Android Studio

**Pull request closed without merging**